hack/deploy-azurite.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
```suggestion
# Copyright (c) 2024 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Reply:
Addressed in [ed3fa51](https://github.com/gardener/etcd-druid/pull/753/commits/ed3fa51b45bc668abd986f5653996ed03aaff3ef).